### ISSUE TITLE
Implement Drag and Drop

### ISSUE THREAD

**Issue by Gronkdalonka** [Gronkdalonka/SportDuell#2]
Drag and Drop still needs to be implemented, e.g. the onDrop stuff needs to be implemented into the calendar etc.

**Comment by Gronkdalonka** [comment]
..so we can drag the different activities into the calendar...

**Comment by Gronkdalonka** [comment]
OR: Other thougt: 

What about we use the calendar just as a schedule and on clicking an event the event will open, and inside this event lies the real training schedule where you can drag and drop the training stuff.
This way we don't need to implement extra "drop-in"-events into the calendar.
All training sessions can be stored seperately in the database and upon opening the calendar we wouldn't need to load so much data only the time and name of the events.
I think this is like it was supposed to be.

So to sum this up: seperate view and controller for Training sessions, implemented trough the state provider so we can force a login to view / edit.

@SchwarzSkills 
@hstender

**Comment by SchwarzSkills** [comment]
Another thought:

If the drag'n'drop stuff gets too complicated, we could also just open another window, where the person enters the data needed, and feed that into the calendar.

**Comment by Gronkdalonka** [comment]
I think we'll ditch this for now, no time left
